tests/dummy/app/templates/snippets/the-list-10.hbs

```diff
@@ -0,0 +1,11 @@
+{{#power-select
+  options=names
+  renderInPlace=true
+  selected=name345
+  onchange=(action (mut name345))
+  as |name|
+}}
+  {{name}}
+  <a onmouseup={{action "stopPropagation"}} 
+    onclick={{action "removeName" name}}>x</a>
```

Comment:
use `&times;` instead of `x`. It looks nicer

Reply:
Noted Miguel, gonna make both changes pretty soon, do you know why the travis build is failing ? will that be a problem ?